src/Controller/AnimalController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class AnimalController extends AbstractController
```

Comment:
try to not extend the `AbstractController`, as it brings along useless functionality which I would like for us to implement manually in the future